airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/kotlin/io/airbyte/cdk/integrations/debezium/CdcSourceTest.kt

```diff
@@ -318,6 +344,16 @@ abstract class CdcSourceTest<S : Source, T : TestDatabase<*, T, *>> {
             .collect(Collectors.toList())
     }
 
+    protected fun extractTraceMessages(
+        messages: List<AirbyteMessage>
+    ): MutableList<io.airbyte.protocol.models.v0.AirbyteTraceMessage>? {
+        return messages
+            .stream()
```

Comment:
NIT: `messages.filter{it.type == AirbyteMessage.Type.TRACE}.map{it.trace}.toList()`

Reply:
Very cool. I haven't used this before.